### PR TITLE
Fix/dedicated expired tasks

### PR DESCRIPTION
**Pull request by JeremyDec**
MANAGER-2179

### CODE REVIEW

File: client/app/dedicated/server/dedicated-server.html [hunk 1]
```diff
@@ -1,112 +1,130 @@
...
-                 role="alert"
-                 data-ng-if="server.isExpired"
-                 data-translate="common_expired">
-            </div>
-            <div class="alert alert-danger"
-                 role="alert"
-                 data-ng-if="server.state == 'HACKED' || server.state == 'HACKED_BLOCKED'"
-                 data-translate="{{ 'server_configuration_state_info_' + server.state }}">
-            </div>
+        <oui-header-tabs>
```
Comment (JeremyDec) [inline client/app/dedicated/server/dedicated-server.html]: Refactor only concerns this new headers tabs

**Comment by ovh-cds** [comment]
CDS Report build-manager#968.0 ■   
* Stage 1
  * Build dependencies and inject Sentry ■

File: client/app/dedicated/server/dashboard/dedicated-server-dashboard.routes.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import template from './dedicated-server-dashboard.html';
+
+angular.module('App').config(($stateProvider) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/dashboard/dedicated-server-dashboard.routes.js]: We encourage you to add explicit ngInject wherever you can.

```suggestion
angular.module('App').config(/* @ngInject */($stateProvider) => {
```

File: client/app/dedicated/server/display-name/display-name.routes.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 angular.module('App')
   .config(($stateProvider) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/display-name/display-name.routes.js]:
```suggestion
  .config(/* @ngInject */($stateProvider) => {
```

File: client/app/dedicated/server/dns/dedicated-server-dns.routes.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import template from './dedicated-server-dns.html';
+
+angular.module('App').config(($stateProvider) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/dns/dedicated-server-dns.routes.js]:
```suggestion
angular.module('App').config(/* @ngInject */($stateProvider) => {
```

File: client/app/dedicated/server/firewall/dedicated-server-firewall.routes.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import template from './dedicated-server-firewall.html';
+
+angular.module('App').config(($stateProvider) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/firewall/dedicated-server-firewall.routes.js]:
```suggestion
angular.module('App').config(/* @ngInject */($stateProvider) => {
```

File: client/app/dedicated/server/ftp-backup/dedicated-server-ftp-backup.routes.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import template from './dedicated-server-ftp-backup.html';
+
+angular.module('App').config(($stateProvider) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/ftp-backup/dedicated-server-ftp-backup.routes.js]:
```suggestion
angular.module('App').config(/* @ngInject */($stateProvider) => {
```

File: client/app/dedicated/server/intervention/dedicated-server-intervention.routes.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import template from './dedicated-server-intervention.html';
+
+angular.module('App').config(($stateProvider) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/intervention/dedicated-server-intervention.routes.js]:
```suggestion
angular.module('App').config(/* @ngInject */($stateProvider) => {
```

File: client/app/dedicated/server/ipmi/dedicated-server-ipmi.routes.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
+import template from './dedicated-server-ipmi.html';
+
+angular.module('App').config(($stateProvider) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/ipmi/dedicated-server-ipmi.routes.js]:
```suggestion
angular.module('App').config(/* @ngInject */($stateProvider) => {
```

File: client/app/dedicated/server/monitoring/update/dedicated-server-monitoring-update.routes.js [hunk 9]
```diff
@@ -1,6 +1,6 @@
 angular.module('App')
   .config(($stateProvider) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/monitoring/update/dedicated-server-monitoring-update.routes.js]:
```suggestion
  .config(/* @ngInject */($stateProvider) => {
```

File: client/app/dedicated/server/ovh-tasks/dedicated-server-ovh-tasks.routes.js [hunk 10]
```diff
@@ -1,7 +1,7 @@
 angular
   .module('App')
   .config(($stateProvider) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/ovh-tasks/dedicated-server-ovh-tasks.routes.js]:
```suggestion
  .config(/* @ngInject */($stateProvider) => {
```

File: client/app/dedicated/server/reboot/dedicated-server-reboot.routes.js [hunk 11]
```diff
@@ -1,6 +1,6 @@
 angular.module('App')
   .config(($stateProvider) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/reboot/dedicated-server-reboot.routes.js]:
```suggestion
  .config(/* @ngInject */($stateProvider) => {
```

File: client/app/dedicated/server/task/dedicated-server-task.routes.js [hunk 12]
```diff
@@ -0,0 +1,13 @@
+import template from './dedicated-server-task.html';
+
+angular.module('App').config(($stateProvider) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/task/dedicated-server-task.routes.js]:
```suggestion
angular.module('App').config(/* @ngInject */($stateProvider) => {
```

File: client/app/dedicated/server/usb-storage/dedicated-server-usb-storage.routes.js [hunk 13]
```diff
@@ -0,0 +1,13 @@
+import template from './dedicated-server-usb-storage.html';
+
+angular.module('App').config(($stateProvider) => {
```
Comment (antleblanc) [inline client/app/dedicated/server/usb-storage/dedicated-server-usb-storage.routes.js]:
```suggestion
angular.module('App').config(/* @ngInject */($stateProvider) => {
```

File: client/app/dedicated/server/dashboard/dedicated-server-dashboard.routes.js (hunk 2 above)
Comment (JeremyDec) [reply client/app/dedicated/server/dashboard/dedicated-server-dashboard.routes.js]: Hmm I forgot these ones...

Comment (JeremyDec) [reply client/app/dedicated/server/dashboard/dedicated-server-dashboard.routes.js]: It's done

File: client/app/dedicated/server/dns/dedicated-server-dns.routes.js [hunk 14]
```diff
@@ -0,0 +1,13 @@
+import template from './dedicated-server-dns.html';
```
Comment (FredericEspiau) [inline client/app/dedicated/server/dns/dedicated-server-dns.routes.js]: Looks like all those routes could be generated automatically ;)

Comment (JeremyDec) [reply client/app/dedicated/server/dns/dedicated-server-dns.routes.js]: Hmm why not, but how to do it ?

**Pull request merged**